lib/google/directions/request.rb

```diff
@@ -22,11 +24,23 @@ def get(params)
       private
 
       def parse_params
-        {
+        parsed_params = {
           sensor:      params[:sensor] || false,
           origin:      params[:origin] || missing(:origin),
-          destination: params[:destination] || missing(:destination)
+          destination: params[:destination] || missing(:destination),
```

Comment:
In ruby is optional, so not a real syntax error, but we always prefer to avoid it.

Done in db46fa8